src/app/services/map.service.ts

```diff
@@ -698,7 +712,7 @@ export class MapService {
       'id': 'way_fill', 'type': 'fill', 'minzoom': minzoom, 'source': 'ways',
       'paint': { 'fill-color': { 'property': 'hexColor', 'type': 'identity' }, 'fill-opacity': 0.3 }
       // ,'filter': ['all']
-      , 'filter': ['all', ['match', ["geometry-type"], ['Polygon', 'MultiPolygon' ], true, false]]
+      , 'filter': ['all', ['match', ["geometry-type"], ['Polygon', 'MultiPolygon' ], true]]

```

Comment:
@Binnette Do you know what the last value, the `false` that I have removed, is good for? I'm wondering because the typing system says there should not be a fourth value. Same applies to the next three changes.

Reply:
https://maplibre.org/maplibre-gl-js-docs/style-spec/expressions/#match
this is the fallback value for "match" expression. The result is probably the same if we remove it

Reply:
I just tested, we must have 4 parameters. Otherwise we get an error

Error: layers.way_line_changed.filter[1]: Expected at least 4 arguments, but found only 3.

You can simply test with an "npm start".

Reply:
Thanks for the review. Indeed you're right and I needed to revert those changes. It seems the typing information in the lib is wrong which might indeed be the case because it's currently [hardcoded](https://github.com/maplibre/maplibre-gl-js/blob/main/build/generate-style-spec.ts#L151) and not derived from the code.

Reply:
Oh yes, right. Thank you.  It's not optimal...
Maybe we can use "[patch-package](https://www.npmjs.com/package/patch-package)" while waiting to fix the package from Maplibre?

Reply:
Maplibre is a fork of Mapbox GL JS.
For Expression, they simply use :  type Expression = [ExpressionName, ...any[]];

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ff48d60936cbd834e68832e17de50e33d7c0031e/types/mapbox-gl/index.d.ts#L175